src/dataEntryApp/views/subjectDashBoard/components/subjectDashboardProgramView.js

```diff
@@ -57,7 +53,7 @@ const useStyles = makeStyles(theme => ({
 
 const ProgramView = ({ programData }) => {
   const classes = useStyles();
-  const [expanded, setExpanded] = React.useState(false);
+  const [expandedPanel, setExpanded] = React.useState(false);
```

Comment:
@RashmiDavandePsl This is still not addressing the issue that I pointed out that you are storing two different types in same variable - boolean and string. Let me know over Gitter if it is not clear how to fix it.

Reply:
@hithacker Will you please elaborate. As I'm not able to understand what you are trying to say.

Reply:
@RashmiDavandePsl You are initializing this state variable as a boolean(false). But you are also storing a string "panel1" or "panel2" when you are calling handleChange("panel1"). It is good to have the variable only hold data of specific type. But here we are storing a boolean and a string inside the same variable.